roles/coco_data_pvc_deploy/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Create PVC for storing the coco dataset
+  command: 
+    oc create -f {{ nvidiapvc }}
```

Comment:
same comment as below, use `oc apply` to avoid failure if the resource already exist,
or maybe, test if it exists before creating it (usually  `oc apply` is simpler, but I don't know how the PVC will behave on update)